eos-s3/requirements.txt

```diff
@@ -1,4 +1,4 @@
 python-constraint
 serial
 git+https://github.com/QuickLogic-Corp/quicklogic-fasm@aaf4c314a165b6185b0983019d8aae4d0d4db6cb
-https://github.com/chipsalliance/f4pga/archive/main.zip#subdirectory=f4pga
+git+https://github.com/antmicro/f4pga.git@pcza/ql-new-flow#egg=f4pga&subdirectory=f4pga
```

Comment:
Can you please change this to use the upstream?

Reply:
Done